src/modules/filters/components/FiltersFetcher/FiltersFetcher.jsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 import { childrenPropType } from '@ackee/lucas';
 import useFetchFilters from '../../hooks/useFetchFilters';
-
 const FilterFetcher = ({ children }) => {
     const { success, inProgress } = useFetchFilters();
 
-    return <>{inProgress || !success ? <h1>Načítam</h1> : children}</>;
+    return <>{inProgress || !success ? null : children}</>;
```

Comment:
What about the new Loader? :)

Reply:
It just didnt feel right to have two loaders on one page.
Should I still add it here ?

Reply:
yess, the loader should be just in this area of the page
every api request could take different amount of time so different components have their own loaders :)

Reply:
ideally filters loader should be visible in the top area of the page where loaders are and reports loader underneath in the list area

you can do it in some new MR